internal/control_test_engine/gnb/context/context.go

```diff
@@ -422,29 +422,19 @@ func (gnb *GNBContext) GetMccAndMnc() (string, string) {
 }
 
 func (gnb *GNBContext) GetMccAndMncInOctets() []byte {
```

Comment:
GetMccAndMncInOctets() are the same for UE and GNB, woudn't it be better to make a common function?